Archipelago.HollowKnight/Archipelago.cs

```diff
@@ -17,6 +17,7 @@
 using ItemChanger.Tags;
 using Modding;
 using UnityEngine;
+using UnityEngine.SceneManagement;
```

Comment:
Probably no longer necessary

Reply:
Oops, that was left in when I was trying to figure out how to actually change to the menu scene.  Definitely not needed.